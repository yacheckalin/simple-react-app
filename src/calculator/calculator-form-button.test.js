import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import Button from "./calculator-form-button";

describe("<Button />", () => {
  test("renders and displays properly", () => {
    const wrapper = shallow(<Button>Submit</Button>);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
