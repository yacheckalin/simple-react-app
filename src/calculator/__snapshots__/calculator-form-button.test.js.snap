// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders and displays properly 1`] = `
<styled.button
  onClick={[Function]}
>
  Submit
</styled.button>
`;
